frontend/add-client/confirm.component.tsx

```diff
@@ -64,8 +75,47 @@ export default function Confirm(props: StepComponentProps) {
     }
   }, [saving, props.clientState]);
 
+  const c = props.clientState;
+
+  const viewableClient: SingleClient = {
```

Comment:
The add client and view client have slightly different representations of the client object. This is the conversion from one to the other.